### ISSUE TITLE
"Type Dict cannot be instantiated" error when executing code line from readme

### ISSUE THREAD

**Issue by jn73** [santinic/pampy#34]
I get this error when i try to match a Dict[str, int] :

I cant find any imports for Dict but assumed it was from typing?

```
In [1]: from pampy import match, _

In [2]: from typing import Dict

In [3]: match({"a": 1, "b": 2}, Dict[str, int], lambda x: x)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-e5c0ae675d5a> in <module>
----> 1 match({"a": 1, "b": 2}, Dict[str, int], lambda x: x)

~/.virtualenvs/sqlparse/lib/python3.7/site-packages/pampy/pampy.py in match(var, default, strict, *args)
    176
    177     for patt, action in pairs:
--> 178         matched_as_value, args = match_value(patt, var)
    179
    180         if matched_as_value:

~/.virtualenvs/sqlparse/lib/python3.7/site-packages/pampy/pampy.py in match_value(pattern, value)
     45         return match_dict(pattern, value)
     46     elif callable(pattern):
---> 47         return_value = pattern(value)
     48         if return_value is True:
     49             return True, [value]

/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/typing.py in __call__(self, *args, **kwargs)
    666     def __call__(self, *args, **kwargs):
    667         if not self._inst:
--> 668             raise TypeError(f"Type {self._name} cannot be instantiated; "
    669                             f"use {self._name.lower()}() instead")
    670         result = self.__origin__(*args, **kwargs)

TypeError: Type Dict cannot be instantiated; use dict() instead
```

**Comment by jn73** [comment]
same with:

`match(1, Union[int, str], lambda s: s)`

=> TypeError: Cannot instantiate typing.Union

**Comment by jn73** [comment]
version: Python 3.7.3

**Comment by jn73** [comment]
Looks like there's some issue with the version (0.2.1) i have installed (pip install pampy).

When i did a 'pip install -e .' from the latest source it works.

**Comment by miaoever** [comment]
Same issue here for the Python 3.7.3, installed by: pip install pampy

**Comment by aambrozkiewicz** [comment]
+1 same here

**Comment by santinic** [comment]
Just published v0.3
